golang-deploy/Dockerfile

```diff
@@ -3,11 +3,26 @@ ARG DOCKERIZE_VERSION="v0.6.1"
 
 USER root
 RUN apt-get update && \
-    apt-get install -y git-crypt make sshpass awscli tree
+    apt-get install -y \
+        git-crypt \
+        make \
+        sshpass \
+        awscli \
+        tree \
+        nodejs npm
```

Comment:
```suggestion
        nodejs \
        npm
```

Reply:
that `nodejs npm` was intended in one line cause it is a "group" 🤷‍♂️  but no strong opinion on that 😅